testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveDatabricksHttpMetastore.java

```diff
@@ -0,0 +1,80 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.product.hive;
+
+import com.google.common.collect.ImmutableList;
+import io.trino.tempto.BeforeMethodWithContext;
+import io.trino.tempto.ProductTest;
+import io.trino.testng.services.Flaky;
+import io.trino.tests.product.utils.DatabricksUnityCatalogUtils;
+import org.testng.annotations.Test;
+
+import java.util.Locale;
+
+import static io.trino.tempto.assertions.QueryAssert.Row.row;
+import static io.trino.testing.TestingNames.randomNameSuffix;
+import static io.trino.tests.product.TestGroups.HIVE_DATABRICKS_HTTP_METASTORE;
+import static io.trino.tests.product.TestGroups.PROFILE_SPECIFIC_TESTS;
+import static io.trino.tests.product.deltalake.util.DeltaLakeTestUtils.DATABRICKS_COMMUNICATION_FAILURE_ISSUE;
+import static io.trino.tests.product.deltalake.util.DeltaLakeTestUtils.DATABRICKS_COMMUNICATION_FAILURE_MATCH;
+import static io.trino.tests.product.utils.DatabricksUnityCatalogUtils.getCreateSchemaWithLocationQuery;
+import static io.trino.tests.product.utils.DatabricksUnityCatalogUtils.getRandomPathInExternalLocation;
+import static io.trino.tests.product.utils.DatabricksUnityCatalogUtils.getS3BucketName;
+import static io.trino.tests.product.utils.QueryExecutors.onDelta;
+import static io.trino.tests.product.utils.QueryExecutors.onTrino;
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestHiveDatabricksHttpMetastore
+        extends ProductTest
+{
+    private String unityCatalogCatalogName;
+    protected String bucketName;
+
+    @BeforeMethodWithContext
+    public void setUp()
+    {
+        bucketName = requireNonNull(System.getenv("S3_BUCKET"), "Environment variable not set: S3_BUCKET");
+        unityCatalogCatalogName = DatabricksUnityCatalogUtils.getTestCatalogName();
+    }
+
+    @Test(groups = {HIVE_DATABRICKS_HTTP_METASTORE, PROFILE_SPECIFIC_TESTS})
+    @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
+    public void testBasicHiveOperations()
+    {
+        String schemaName = "testBasicHiveOperations_" + randomNameSuffix();
+        String tableName = schemaName + "." + "testTable";
+        String tableLocation = getRandomPathInExternalLocation(tableName);
+        onDelta().executeQuery(getCreateSchemaWithLocationQuery(schemaName));
+        onDelta().executeQuery(format(
+                "CREATE TABLE %s.%s (c1 int, c2 string) " +
+                        "USING PARQUET " +
+                        "LOCATION 's3://%s/%s'", unityCatalogCatalogName, tableName, getS3BucketName(), tableLocation));
+
+        onDelta().executeQuery(
+                format(
+                        "INSERT INTO %s.%s values (1, 'one')", unityCatalogCatalogName, tableName));
```

Comment:
Can we insert via `onTrino()` as well in Hive?

Reply:
I tried this and realized Trino does not support inserting into non-managed tables.

Reply:
Try out adding the following setting to `hive.properties`

https://github.com/trinodb/trino/blob/b95462011872c6261862651e64d9a5b4182d4c59/plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveConfig.java#L923-L929

Reply:
yeah after I added that config, it made progress beyond that and exposed the issue where it needs `get_table_statistics_req` to be implemented. This is a known issue and needs to be fixed on the Databricks service side.